sampleconfig/orderer.yaml

```diff
@@ -334,6 +334,35 @@ Metrics:
       # The prefix is prepended to all emitted statsd metrics
       Prefix:
 
+################################################################################
+#
+#   Admin Configuration
+#
+#   - This configures the admin server endpoint for the orderer
+#
+################################################################################
+Operations:
```

Comment:
I'm confused, why do you have another operations endpoint, [exactly like the existing one](https://github.com/hyperledger/fabric/blob/master/sampleconfig/orderer.yaml#L288-L309)?

Aren't both endpoints supposed to be used by users possessing local admin credentials? Why do you need a separate port, isn't it possible to use the existing one?

Reply:
My understanding is that the role using this admin endpoint to join/leave channels and the role using the operations endpoint to access metrics, change log levels. etc. are distinct and should be configurable separately.

Reply:
Yes, the key here is that the identity performing operations is not a Fabric admin, it's a system admin.  We would not want the identity used to perform health check, pull metrics, or modify log level to be the same identity responsible for joining/leaving channels.  This is something that @sykesm brought up during the RFC review process and a new port is the simplest option going forward.  In the future, the plan is to bring some non-MTLS authentication mechanism in, but that it outside the scope of the channel participation effort.

Reply:
> Yes, the key here is that the identity performing operations is not a Fabric admin, it's a system admin. We would not want the identity used to perform health check, pull metrics, or modify log level to be the same identity responsible for joining/leaving channels.

You can still accomplish this with a single port. You can allow both types of connections and then classify the client by its certificate.

Reply:
Ah, thanks for clarifying, Yoav. Definitely overlooked this. Fixed.